mmpose/visualization/local_visualizer.py

```diff
@@ -243,6 +248,20 @@ def _draw_instances_kpts(self,
             else:
                 keypoints_visible = [np.ones(len(kpts)) for kpts in keypoints]
 
+            if skeleton_style == 'openpose':
+                neck = (keypoints[:, 5] + keypoints[:, 6]) / 2
+                neck_score = (scores[:, 5] + scores[:, 6]) / 2
+                keypoints = np.insert(keypoints, 17, neck, axis=1)
+                scores = np.insert(scores, 17, neck_score, axis=1)
+
+                original_cols = [
+                    17, 6, 8, 10, 7, 9, 12, 14, 16, 13, 15, 2, 1, 4, 3
+                ]
+                new_cols = [1, 2, 3, 4, 6, 7, 8, 9, 10, 12, 13, 14, 15, 16, 17]
+                keypoints[:, new_cols] = keypoints[:, original_cols]
+                scores[:, new_cols] = scores[:, original_cols]
+                keypoints_visible = [np.ones(len(kpts)) for kpts in keypoints]
```

Comment:
When visualizing images from GT in openpose-style, `keypoints_visible` should not be np.ones() for all keypoints

Reply:
Does `keypoints_visible` to the `joint_weights` in coco_openpose.py?Here I refer to https://github.com/open-mmlab/mmpose/pull/2115/files/9630c10a895bec012cd44b2a882f8c1c143d9318#diff-070475f91f5ca244130def9db60fe354566df801e1a23aa33d3b2060c9c8515cL244,I don't understand `keypoints_visible`

Reply:
just refer to L246-L249:
```python
if 'keypoints_visible' in instances:
    keypoints_visible = instances.keypoints_visible
else:
    keypoints_visible = [np.ones(len(kpts)) for kpts in keypoints]
```
`joint_weights` is used in loss calculation.

Reply:
> When visualizing images from GT in openpose-style, `keypoints_visible` should not be np.ones() for all keypoints

Thank you very much for your explanation, I understand now!